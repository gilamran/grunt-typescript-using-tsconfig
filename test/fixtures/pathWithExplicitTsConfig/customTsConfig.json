{
  "compilerOptions": {
    "listFiles": true,
    "target": "es5",
    "module": "amd",
    "removeComments": false,
    "declaration": true,
    "sourceMap": true,
    "outDir": "./"
  },
  "filesGlob": [
    "**/*.ts"
  ],
  "files": [
  ]
}
