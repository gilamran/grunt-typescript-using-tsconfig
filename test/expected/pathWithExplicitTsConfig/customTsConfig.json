{
  "compilerOptions": {
    "listFiles": true,
    "target": "es5",
    "module": "amd",
    "removeComments": false,
    "declaration": true,
    "sourceMap": true,
    "outDir": "./"
  },
  "filesGlob": [
    "**/*.ts"
  ],
  "files": [
    "dummy1.ts",
    "dummy2.ts"
  ]
}
